packages/manager/apps/web/client/app/domain/dns-modify/components/internal-config-radio.html

```diff
@@ -0,0 +1,60 @@
+<oui-message data-ng-if="!$ctrl.isZone" data-type="info">
+    <span>
+        {{ ::"domain_dns_default_config_activate_zone_message" | translate }}
+    </span>
+    <a data-ng-href="{{::$ctrl.links.zoneActivate}}">
+        {{ ::'domain_dns_config_activate_zone_message_link' | translate }}
+    </a>
+</oui-message>
+
+<oui-radio
+    disabled="!$ctrl.isZone"
+    name="dns_configuration_types"
+    value="::$ctrl.constants.CONFIGURATION_TYPES.INTERNAL"
+    data-model="$ctrl.selectedConfigurationType"
+    data-on-change="$ctrl.onConfigurationTypeChange()"
+>
+    <oui-radio-label>
+        <span>
+            {{ ::"domain_dns_use_default_ovh_name_servers_title" | translate }}
+        </span>
+        <span
+            data-ng-if="$ctrl.currentConfigurationType !== $ctrl.constants.CONFIGURATION_TYPES.INTERNAL"
+            class="oui-badge oui-badge_info oui-badge_s"
+        >
+            {{ ::"domain_dns_recommended_label" | translate }}
+        </span>
+        <span
+            data-ng-if="$ctrl.currentConfigurationType === $ctrl.constants.CONFIGURATION_TYPES.INTERNAL"
+            class="oui-badge oui-badge_success oui-badge_s"
+        >
+            {{ ::"domain_dns_active_label" | translate }}
+        </span>
+    </oui-radio-label>
+    <oui-radio-description>
+        <span>
+            {{ ::"domain_dns_use_default_ovh_name_servers_description" |
+            translate }}
+        </span>
+        <div
+            data-ng-if="$ctrl.selectedConfigurationType === $ctrl.constants.CONFIGURATION_TYPES.INTERNAL"
+        >
+            <oui-message
+                data-ng-if="alerts.update"
+                data-type="error"
+                class="mt-3"
+            >
+                <span
+                    data-ng-bind-html="alerts.update.message | translate:alerts.update.data"
```

Comment:
as it is a translated text inside your span, use instead data-translate and data-translate-values